src/main/java/org/folio/cql2rmapi/CQLParserForRMAPI.java

```diff
@@ -274,9 +278,9 @@ String buildRMAPIQuery(int limit, int pageOffsetRMAPI) throws QueryValidationExc
         builder.append("&resourcetype=" + filterValue);
       }
 
+      final String queryParam = "&selection=";
```

Comment:
Can we please move this declaration to the beginning of the method and also rename the variable to be more specific , something like: selectionQueryParam for readability?

Reply:
I'm not fan of moving variables away from where they are being used. But, I'll rename the variable to make it more clear what the intent is.

Reply:
Now that I think about it, I'll just make it a constant. It will be in every query, so we may as well have it defined at the class level.

Reply:
Not sure about that...we would want to confine variables to their scope as much as possible.  Since, it is being used only in that method, I would prefer we do not declare it at class level. Also, I am ok with declaring it right above where it is being used but it is not a good practice according to Sun --> Section 6.3 --> :Phttp://www.oracle.com/technetwork/java/javase/documentation/codeconventions-141270.html

Reply:
Oooh! Google agrees with me: https://google.github.io/styleguide/javaguide.html#s4.8.2-variable-declarations